app/javascript/components/admin/AdminTable.js

```diff
@@ -23,7 +23,8 @@ class AdminTable extends React.Component {
           { label: 'Email', field: 'email', className: 'wrap-words', isSortable: true },
           { label: 'Jurisdiction', field: 'jurisdiction_path', isSortable: true },
           { label: 'Role', field: 'role_title', isSortable: false },
-          { label: 'Status', field: 'is_locked', isSortable: false, options: { true: 'Locked', false: 'Unlocked' } },
+          { label: 'System Access', field: 'is_locked', isSortable: false, options: { true: 'Locked', false: 'Unlocked' } },
```

Comment:
I know this wasn't previously sortable but I think it makes sense for this to also be sortable given that the `Status` column is and it's a related field

Reply:
Fine with me, feel free to make the System Access column sortable as well

Reply:
@sgober I wasn't sure about the best way to achieve this programatically. Let me know what you think: [b8dd3ee403d317e23955cc1ceb8bdfcda6b7ce93](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/b8dd3ee403d317e23955cc1ceb8bdfcda6b7ce93)